packages/dynamite/dynamite/lib/src/helpers/dart_helpers.dart

```diff
@@ -84,6 +86,13 @@ final _dartKeywords = [
   'with',
 ];
 
+const _dartTypes = [
```

Comment:
Why not put them to the other keywords? I don't see any special handling because they are types :woman_shrugging: